CMakeLists.txt

```diff
@@ -5,24 +5,24 @@
 # lowest version supporting C_STANDARD property
 cmake_minimum_required (VERSION 3.1.3)
 
+# Set software information
+project(libsndfile VERSION 1.0.28.0)
+
```

Comment:
I think the final `.0` may be needed for something in the Windows build chain, but should not show up in the non-Windows builds.